arm9/source/utils/gameutil.c

```diff
@@ -1227,24 +1227,22 @@ u32 BuildCiaFromTmdFileBuffered(const char* path_tmd, const char* path_cia, bool
     if (!name_content) return 1; // will not happen
     name_content++;
     
+    bool present[TMD_MAX_CONTENTS];
+    memset(present, true, sizeof(present));
```

Comment:
Is `bool` guaranteed to be 1 byte here? Granted, this will work even if it's not.

Reply:
You're not sure, either, correct? Maybe `bool present[TMD_MAX_CONTENTS] = { true }` would be more sane?

Reply:
It's implementation-defined, although I believe it is 1 byte in the case of devkitARM's environment.

Reply:
If you want, I could flip it to "missing" and initialize it to all zeros instead of explicitly using "true".

Reply:
Okay, suggestion. Use an u8 array instead. Just to be safe. I know, this is again nitpicking, but I don't like the memset when we're not really sure about the size of the elements in the `present` array. Open for discussion.